projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/saved-cart/b2b-saved-cart.e2e-spec.ts

```diff
@@ -100,6 +100,33 @@ context('B2B - Saved Cart', () => {
         it('should be able to save the active cart and view it in the listing page', () => {
           savedCart.saveActiveCart();
         });
+
+        it('should create cart, save it and restore with a clone with default form values twice and then compare row in saved cart listing page', () => {
+          savedCart.saveActiveCart();
+
+          cy.get('cx-saved-cart-list .cx-saved-cart-list-table tr')
+            .first()
+            .find('.cx-saved-cart-list-cart-id > .cx-saved-cart-list-value')
+            .then((cartId) => {
+              const cartCode = cartId.text();
+
+              savedCart.manuallyRestoryCartFromListingPage(cartCode);
```

Comment:
you should assert that you have a cloned saved cart afterwords in the listing page